rust/common/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tvm-common"
+version = "0.1.0"
+authors = ["Ehsan M. Kermani <ehsanmo1367@gmail.com>"]
```

Comment:
I think this might need to become something like "TVM Project <tvm@apache.org>". @tqchen? In any case, it'd be fairer to new rust contributors to keep it open and inclusive.

Reply:
Sure! but is the email registered officially or should wait to see how apache goes? @tqchen

Reply:
#2500